Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -125,6 +125,9 @@ public struct BSONNull: BSONValue, Codable {
 
     public static func from(iterator iter: DocumentIterator) throws -> BSONNull { return BSONNull() }
 
+    /// Initializes a new, empty `BSONNull` instance.
```

Comment:
```suggestion
    /// Initializes a new `BSONNull` instance.
```
nit: I feel like "empty" implies there could be data stored in it, which is never the case.